frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/__mocks__/pipeline-run-mock.ts

```diff
@@ -0,0 +1,132 @@
+const mockPipelineRun = {
+  apiVersion: 'tekton.dev/v1alpha1',
+  kind: 'PipelineRun',
```

Comment:
nit : can we take this values from model, it'll be useful if `PipelineRun` version changes

Reply:
🤔 Wonder if it matters... there is a certain separation between tests and the real world... Probably would cause an annoyance... definitely not opposed to it.